src/types/importCandidate.types.ts

```diff
@@ -10,15 +10,20 @@ import { MediaContributionPeriodicalPublicationInstance } from './publication_ty
 import { ResearchDataPublicationInstance } from './publication_types/researchDataRegistration.types';
 import { MapPublicationInstance } from './publication_types/otherRegistration.types';
 import { ExhibitionPublicationInstance } from './publication_types/exhibitionContent.types';
-import { Journal, Publisher } from './registration.types';
+import { Journal, Publisher, Registration } from './registration.types';
 
 export enum ImportStatus {
   Imported = 'IMPORTED',
   NotImported = 'NOT_IMPORTED',
   NotApplicable = 'NOT_APPLICABLE',
 }
 
-export interface ImportCandidate {
+export interface ImportCandidate extends Omit<Registration, 'type'> {
+  type: 'ImportCandidate';
+  importStatus: ImportStatus;
+}
+
+export interface ExpandedImportCandidate {
   type: 'ImportCandidate';
```

Comment:
Om det er slik at ÈxpandedImportCandidate` også har alle felter som er i `ImportCandidate` (som jeg ville antatt ut fra navnet):
```suggestion
export interface ExpandedImportCandidate extends ImportCandidate {
```

Reply:
Når jeg ser på flere av properties under her, mistenker jeg at jeg ikke har rett i den antakelsen, men vi kan ta det på kontoret over helga :)

Reply:
De to typene er ganske forskjellige, men enig i at ExpandedImportCandidate er egentlig en ImportCandidateSummary, da den har langt færre felter en "vanlig" importCandidate har.

Reply:
Enig med at det gir mening som ImportCandidateSummary